src/components/activity-list/ActivityList.js

```diff
@@ -114,52 +114,58 @@ const ActivityList = ({
     [colors]
   );
 
-  return network === networkTypes.mainnet || sections.length ? (
-    recyclerListView ? (
-      <RecyclerActivityList
-        addCashAvailable={addCashAvailable}
-        header={header}
-        isEmpty={isEmpty}
-        isLoading={isLoading}
-        navigation={navigation}
-        sections={sections}
-      />
-    ) : isEmpty ? (
-      <ActivityListEmptyState>{header}</ActivityListEmptyState>
-    ) : (
-      <SectionList
-        ListFooterComponent={() =>
-          remainingItemsLabel && (
-            <ListFooterComponent
-              label={remainingItemsLabel}
-              onPress={nextPage}
-            />
-          )
-        }
-        ListHeaderComponent={header}
-        alwaysBounceVertical={false}
-        contentContainerStyle={{ paddingBottom: !transactionsCount ? 0 : 40 }}
-        extraData={{
-          hasPendingTransaction,
-          nativeCurrency,
-          pendingTransactionsCount,
-        }}
-        getItemLayout={getItemLayout}
-        initialNumToRender={12}
-        keyExtractor={keyExtractor}
-        removeClippedSubviews
-        renderSectionHeader={renderSectionHeaderWithTheme}
-        sections={sections}
-      />
-    )
-  ) : (
-    <ActivityListEmptyState
-      emoji="👻"
-      label={lang.t('activity_list.empty_state.testnet_label')}
-    >
-      {header}
-    </ActivityListEmptyState>
-  );
+  if (network === networkTypes.mainnet || sections.length) {
```

Comment:
I replaced the nested ternary with the if else statement, seemed much more readable